packages/client.api/src/mapping/DataValueMapping.ts

```diff
@@ -40,7 +40,7 @@ export interface DataValueWireToClient {
  * Map from the DataValue type to the typescript type that we accept
  */
 export interface DataValueClientToWire {
-  attachment: Attachment | AttachmentUpload;
+  attachment: string | AttachmentUpload;
```

Comment:
Now we just let users give us the rid as a `string`, since we deconstruct and send only the string to the wire anyways